backend/objstore/objstore.go

```diff
@@ -63,6 +43,12 @@ func Client() (*minio.Client, error) {
 	})
 }
 
+// Client with environment based configuration. Registers healthchecks automatically. If yo do not want to use healthchecks
+// consider calling CustomClient.
+func DefaultClientFromEnv() (*minio.Client, error) {
+	return Client()
```

Comment:
As you did create a client() function, I think it would be prudent to use that one here instead of the old soon-to-be-deprecated Client()

Reply:
@Ferlonas addressed. please re-review